content/docs/gto/use-cases/model-registry.md

```diff
@@ -0,0 +1,62 @@
+# Machine Learning Model Registry
+
+A **model registry** is a tool to catalog ML models and their versions. Models
+from your data science projects can be discovered, tested, shared, deployed, and
+audited from there. [DVC](https://github.com/iterative/dvc), GTO, and [MLEM]
+enable these capabilities on top of Git, so you can stick to en existing
+software engineering stack. No more divide between ML engineering and
+operations!
+
+[mlem]: /doc
+
+ML model registries give your team key capabilities:
+
+- Collect and organize model [versions] from different sources effectively,
+  preserving their data provenance and lineage information.
+- Share metadata including [metrics and plots][mp] to help use and evaluate
+  models.
+- A standard interface to access all your ML artifacts, from early-stage
+  [experiments] to production-ready models.
```

Comment:
This makes me very curious, have I missed something? Given a mlem model, I will know from which experiment it comes from? how? you store the dvc experiment reference as metadata? Is it explained somewhere?

Reply:
You're right, there is no such thing as link from `.mlem` metafile to the exact experiment. I guess the idea behind this words is that you have a Git repo, and given a commit, you can get your ML experiment (DVC) and model metadata (MLEM) and a signal what is production-ready (GTO). Does it make sense now?

Reply:
Ah ok, thanks for the clarification!

Perhaps this is a point to clarify somewhere. When I think of such tools I personally always imagine to have a separate repo where I store e.g. the mlem models. Why? Mainly because at a certain point I would like to train as part of ci/cd and avoid the creation of new commits to the repo itself as part of the ci/cd. I know it's possible via cml, but I would prefer to avoid the need altogether.

Perhaps it's only me...